tests/test_circulator_reports.py

```diff
@@ -1,103 +1,178 @@
-    with Session(bind=ridesystems_reports.engine, future=True) as session:
+    test_df = {
+        'vehicle': ['Tesla Model S', 'bike', 'legs'],
+        'route': ['Purple', 'Purple', 'Purple'],
+        'stop': ["Webster's house", 'Brian St', 'Quilvio is cool'],
+        'latitude': [123.456789, 123.456789, 123.456789],
+        'longitude': [123.456789, 123.456789, 123.456789],
+        'datetime': [datetime(2022, 1, 1, 12, 0), datetime(2022, 1, 1, 12, 15), datetime(2022, 1, 1, 12, 30)],
+        'entries': [0, 3, 1],
+        'exits': [100, 0, 0],
+        }
+
+    inst = RidesystemReports(conn_str)
+    inst.rs_cls._login.return_value = "Login successful :)"
+    inst.rs_cls.get_ridership.return_value = pd.DataFrame(test_df)
+
+    inst.get_ridership(date(2022, 1, 1), date(2022, 1, 1))
+    with Session(bind=inst.engine, future=True) as session:
         ret = session.query(CirculatorRidership)
-        assert ret.count() > 1000
+        assert ret.count() == 7
 
-
-def test_get_ridership_force(ridesystems_reports):
+@patch('transitstat.circulator.reports.RideSystemsInterface')
+def test_get_ridership_force(mocked_rs_cls, conn_str):
     """Test get_ridership"""
-    ridesystems_reports.get_ridership(date(2021, 5, 1), date(2021, 5, 1), force=True)
-    with Session(bind=ridesystems_reports.engine, future=True) as session:
-        ret = session.query(CirculatorRidership)
-        assert ret.count() > 1000
-
-
-def test_get_ridership_no_force(ridesystems_reports):
-    """Test get_ridership"""
-    ridesystems_reports.get_ridership(date(2021, 5, 1), date(2021, 5, 1), force=False)
-    with Session(bind=ridesystems_reports.engine, future=True) as session:
+    test_df = {
+        'vehicle': ['xx1', 'xx2', 'xx3'],
+        'route': ['xx1', 'xx2', 'xx3'],
+        'stop': ['xx1', 'xx2', 'xx3'],
+        'latitude': [987.654321, 987.654321, 987.654321],
+        'longitude': [987.654321, 987.654321, 987.654321],
+        'datetime': [datetime(2021, 5, 1, 12, 0), datetime(2021, 5, 2, 12, 0), datetime(2021, 5, 3, 12, 0)],
+        'entries': [0, 3, 1],
+        'exits': [100, 0, 0],
+        }
+
+    inst = RidesystemReports(conn_str)
+    inst.rs_cls._login.return_value = "Login successful :)"
+    inst.rs_cls.get_ridership.return_value = pd.DataFrame(test_df)
+
+    inst.get_ridership(date(2022, 1, 1), date(2022, 1, 1), force=True)
+    with Session(bind=inst.engine, future=True) as session:
         ret = session.query(CirculatorRidership)
         assert ret.count() == 4
 
 
+
 def test_get_dates_to_process(ridesystems_reports):
     """Test get_dates_to_process"""
 
     # Test with datetime
     dates = ridesystems_reports.get_dates_to_process(date(2021, 5, 1), date(2021, 5, 4),
-                                                     CirculatorBusRuntimes.starttime, force=False)
+                                                    CirculatorBusRuntimes.starttime, force=False)
```

Comment:
Uh oh... this could be something we look at. I know pycharm wants it lined up the way it was before, and I assume your IDE wanted one less space? Super minor, but we might need to figure out those things in the settings.

Reply:
this was my bad! i didn't run it through flake8. i fixed the indentation in this latest update